src/server/pps/server/worker_rc.go

```diff
@@ -126,6 +126,9 @@ func (a *apiServer) workerPodSpec(options *workerOptions, pipelineInfo *pps.Pipe
 	}, {
 		Name:  client.PPSPipelineNameEnv,
 		Value: pipelineInfo.Pipeline.Name,
+	}, {
+		Name:  "JAEGER_ENDPOINT",
```

Comment:
Could the workers rely on `JAEGER_COLLECTOR_SERVICE_HOST`? I think that would eliminate the need to set any env vars explicitly, and might make deploying Jaeger into an existing cluster easier (just deploy Jaeger and restart the worker pods, and they'll start talking to Jaeger automatically)